mwdb/schema/comment.py

```diff
@@ -18,5 +18,5 @@ class CommentRequestSchema(CommentSchemaBase):
 
 class CommentItemResponseSchema(CommentSchemaBase):
     id = fields.Int(required=True, allow_none=False)
-    author = fields.Str(required=True, allow_none=False, attribute="author_login")
+    author = fields.Str(required=True, default=None, attribute="author_login")
```

Comment:
So we won't remove comments when the user is deleted? How does it look in the mwdb interface?

Reply:
Yes, we don't want to lose information that is potentially valuable.
![image](https://github.com/CERT-Polska/mwdb-core/assets/8720367/33615527-8ea1-4bc3-b171-973e86abc746)